build.sc

```diff
@@ -1559,191 +1650,301 @@ object dev extends MillModule {
   }
 }
 
+
+/** Generates the mill documentation with Antora. */
 object docs extends Module {
+  // This module isn't really a ScalaModule, but we use it to generate
+  // consolidated documentation using the Scaladoc tool.
+  object site extends UnidocModule {
+    def scalaVersion = Deps.scalaVersion
 
-  /** Generates the mill documentation with Antora. */
```

Comment:
I eliminated one layer of nesting; there wasn't anything interesting inside `docs` outside `docs/antora/` anyway